modules/ROOT/pages/mongodb/mongodb-connector-6-0-studio.adoc

```diff
@@ -0,0 +1,100 @@
+= MongoDB Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+To configure a connector in Anypoint Studio follow this process:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+== Install the Connector Using Exchange
+
+. In Studio, create a Mule project.
+. Click the Exchange *(X)* icon in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `MongoDB`.
+. Select the MongoDB Connector and click *Add to project*.
```

Comment:
```suggestion
. Select the Anypoint Connector for MongoDB name and click *Add to project*.
```

Reply:
Changed to *MongoDB Connector* to match the GUI.